src/ekore/operator_matrix_elements/unpolarized/time_like/__init__.py

```diff
@@ -1,15 +1,60 @@
-r"""The polarized, time-like |OME|."""
+r"""The unpolarized, time-like |OME|."""
 
 import numba as nb
+import numpy as np
+
+from . import as1
 
 
 @nb.njit(cache=True)
-def A_non_singlet(_matching_order, _n, _sx, _nf, _L):
-    """Compute the non-singlet |OME|."""
-    raise NotImplementedError("Time-like is not yet implemented")
+def A_non_singlet(matching_order, N, L):
+    r"""Compute the non-singlet |OME|.
+
+    Parameters
+    ----------
+    matching_order : tuple(int, int)
+        perturbative matching order
+    N : complex
+        Mellin moment
+    L : float
+        :math:`\ln(\mu_F^2 / m_h^2)`
+
+    Returns
+    -------
+    A_non_singlet : numpy.ndarray
+        non-singlet |OME|
+
+    """
+    if matching_order[0] > 1:
+        raise Exception("Time-like matching conditions are only known upto NLO")
+    else:
+        A_ns = np.zeros((matching_order[0], 2, 2), np.complex_)
+        A_ns[0] = as1.A_ns()
```

Comment:
```suggestion
    A_ns = np.zeros((matching_order[0], 2, 2), np.complex_)
    A_ns[0] = as1.A_ns()
```
you can simplify a bit (same below)